typings/global.d.ts

```diff
@@ -1,11 +1,16 @@
 // eslint-disable-next-line @typescript-eslint/triple-slash-reference
```

Comment:
optional: we may use multi-line eslint comment here like
`/* eslint-disable @... */`

`/* eslint-enable @... */`

If they don't break triple slash directives.